frontend/src/app/AboutDialog.tsx

```diff
@@ -81,6 +117,75 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
                 : 'Unknown'}
             </Content>
           </Content>
+
+          {/* Component Releases Section */}
+          <Content
+            style={{
+              paddingTop: 48,
+              visibility: loading ? 'hidden' : 'visible',
+            }}
+          >
+            <h4>Installed components</h4>
+            {groupedComponents.length > 0 ? (
+              <>
+                <Content>
+                  <Table
+                    aria-label="Component Releases Table"
+                    data-testid="component-releases-table"
+                  >
+                    <Thead>
+                      <Tr data-testid="table-row-title">
+                        <Th width={10}>
+                          {isRHOAI
+                            ? RhoaiDefaultComponentReleaseName
+                            : OdhDefaultComponentReleaseName}{' '}
+                          Component
+                        </Th>
+                        <Th width={20}>Upstream component</Th>
+                        <Th width={10}>Upstream version</Th>
```

Comment:
Are widths necessary?
Add `modifier="wrap"` to the `Th` to prevent truncation just in case.

Reply:
By removing the unnecessary `<Content>`, it looks like I can remove the width on `Th`. I'll add the `modifier="wrap"` just in case.